core/src/franchise/player/player.service.ts

```diff
@@ -180,61 +180,94 @@ export class PlayerService {
         let player: Player;
 
         try {
-            const user = this.userRepository.create({});
+            const mlePlayer = await this.mle_playerRepository.findOne({where: {mleid} });
+
+            if (mlePlayer)  {
+                const bridge = await this.ptpRepo.findOneOrFail({where: {mledPlayerId: mlePlayer.id} });
```

Comment:
Why would `mlePlayer` exist if this function is to intake new members?

Reply:
It's an upsert.